raiden/network/utils.py

```diff
@@ -63,10 +63,12 @@ def _unused_ports(initial_port: Optional[int]) -> Iterable[Port]:
         initial_port = initial_port or 27854
 
         for port in count(initial_port):
+            # Because it is not know which interface the socket will bind to,
+            # if there is any socket in the target port it must be skiped.
             connect_using_port = (
                 conn
                 for conn in psutil.net_connections()
-                if hasattr(conn, "laddr") and conn.laddr[0] == LOOPBACK and conn.laddr[1] == port
```

Comment:
Otherwise port lookup does not work and running two instances fails. E.g. this wouldn't work:
`python -m raiden smoketest` with `--eth-client parity` and `--eth-client geth`